recipe/meta.yaml

```diff
@@ -33,8 +33,7 @@ requirements:
     - python-dateutil
     - myproxyclient
     - esgf-pyclient
-    # Will need this later for the 1.2.0 release, leaving commended for now
-    # - podaacpy >= 1.4.0
+    - podaacpy
```

Comment:
Not required but just in case, I recommend using `podaacpy >=1.9.0` here given incompatibilities with older versions.